ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -369,13 +381,19 @@ public void onNext(InstallSnapshotReplyProto reply) {
 
       switch (reply.getResult()) {
         case SUCCESS:
```

Comment:
Do we need to update commitIndex and nextIndex here as well?

Reply:
Yes but I am not sure what all needs to be fixed in the regular InstallSnapshot. I can update the snapshotIndex, commitIndex and nextIndex here for now.

Reply:
Yes..Makes sense to do it in a separate jira.